src/sort.c

```diff
@@ -189,7 +189,7 @@ int sortCompare(const void *s1, const void *s2) {
 
 /* The SORT command is the most complex command in Redis. Warning: this code
  * is optimized for speed and a bit less for readability */
-void sortCommand(client *c) {
+void sortCommandGeneric(client *c, bool noStore) {
```

Comment:
Yeah, this should use int. Also either no store or no_store are idiomatic variable names.